### PR TITLE
My history page

### PR DESCRIPTION
**Pull request by Tasadan-Filip**

### CODE REVIEW

File: src/app/components/sidenav/sidenav.component.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ export class SidenavComponent implements OnInit {
         { title: "All Events", name: "calendar_today", route: RouteValues.ALL_EVENTS },
         { title: "Joined Events", name: "list_alt", route: RouteValues.JOINED_EVENTS },
         { title: "Hosted Events", name: "people", route: "" },
-        { title: "My history", name: "history", route: "" },
+        { title: "My history", name: "history", route: "my-history" },
```
Comment (larisaChh) [inline src/app/components/sidenav/sidenav.component.ts]: Use the route constant for this

File: src/app/components/my-history/my-history.component.ts [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  };
+
+  constructor(private eventService: EventService, private categoryService: CategoryService, private appStateService: AppStateService, private attendanceService: AttendanceService, private router: Router, private sortDataService: HelperService) {}
+
+  ngOnInit(): void {
+    this.currentUserId = this.appStateService.getCurrentUserId();
+    let filterData: FiltersData = {
+      startingDate: new Date("1900-01-01"),
+      endingDate: new Date(),
+      AttendingId: this.currentUserId
```
Comment (meda-racasan) [inline src/app/components/my-history/my-history.component.ts]: Why is this property starting with a capital letter?

File: src/app/services/http.service.ts [hunk 3]
```diff
@@ -27,6 +27,12 @@ export class HttpService {
     return this.httpClient.get<T>(requestUrl, { headers: this.headers });
   }
 
+  getByTwo<T>(id1: number, id2: number, endpoint: string): Observable<T> {
```
Comment (meda-racasan) [inline src/app/services/http.service.ts]: `getByTwo` is not a suggestive name for a function.

File: src/app/models/filters-data.ts [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
-    startDate:Date,
-    endDate?:Date,
+    startingDate:Date,
+    endingDate?:Date,
     categories?:number[],
     hiddenIfFee?:boolean,
     hiddenIfStarted?:boolean,
-    isHostId?:number,
-    isAttendingId?:number,
+    HostId?:number,
```
Comment (meda-racasan) [inline src/app/models/filters-data.ts]: Why are these properties starting with a capital letter?

**Pull request merged**